src/main/java/duke/Find.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class Find extends Task {
+    public String target;
+    ArrayList<String> list = new ArrayList<>();
+
+    public Find(String target, ArrayList<String> list) {
+        super(target);
+        this.target = target;
+        for (int k = 0; k < list.size(); k++) {
+            if (list.get(k).contains(target)) {
+                this.list.add(list.get(k));
+            }
+        }
+    }
+
+    /**
+     * Checks if the command should be added to the list.
+     *
+     * @return boolean
+     */
+    @Override
+    public boolean AddToList() {
```

Comment:
Perhaps you can name your method to follow camelCase instead? I noticed the same issue in several other places too.

```Suggestion
    public boolean addToList() {
```